packages/fast-components-react-msft/src/badge/badge.props.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import {
+    BadgeClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-msft";
+
+export enum BadgeAppearance {
+    accent = "accent",
+    highlight = "highlight",
+    lowlight = "lowlight",
+    lightweight = "lightweight",
+}
+
+export enum BadgeSize {
+    small = "small",
+    large = "large",
+}
+
+export interface BadgeManagedClasses extends ManagedClasses<BadgeClassNameContract> {}
+/* tslint:disable-next-line:no-empty-interface */
+export interface BadgeUnhandledProps {}
```

Comment:
This should extend the correct React interface

Reply:
Fixed.